packages/fxa-admin-panel/src/components/AccountSearch/Subscription/index.tsx

```diff
@@ -0,0 +1,72 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import dateFormat from 'dateformat';
+import { MozSubscription } from 'fxa-admin-server/src/graphql';
+import LinkExternal from 'fxa-react/components/LinkExternal';
+import { ReactComponent as IconExternalLink } from '../../../images/icon-external-link.svg';
+import { DATE_FORMAT } from '../Account';
+
+const Subscription = ({
+  created,
+  currentPeriodEnd,
+  currentPeriodStart,
+  cancelAtPeriodEnd,
+  endAt,
+  latestInvoice,
+  planId,
+  productName,
+  productId,
+  status,
+  subscriptionId,
+}: MozSubscription) => {
+  return (
+    <ul className="account-border-info">
+      <li className="account-li">
+        Product name: <span>{productName}</span>
+      </li>
+      <li className="account-li">
+        Status: <span>{status}</span>
+      </li>
+      {created && (
```

Comment:
Random nitpick. If created isn't defined can we indicate this rather than just hiding it? Since this is used by support personnel, there's a small chance the fact that created is undefined could be important. I'd rather we be explicit about the data's state than not showing things.
 
Possibly we should also just hide it... but this came up in Mill's code review as well, and the consensus there was to show the field but indicate it is missing. Ultimately I think it should just be consistent.

Reply:
Good call, I agree.